Sources/Datadog/Tracing/TracingV2Configuration.swift

```diff
@@ -49,3 +49,32 @@ internal struct TracingRequestBuilder: FeatureRequestBuilder {
         return builder.uploadRequest(with: data)
     }
 }
+
+internal struct TracingMessageReceiver: FeatureMessageReceiver {
+    /// Process messages receives from the bus.
+    ///
+    /// - Parameters:
+    ///   - message: The Feature message
+    ///   - core: The core from which the message is transmitted.
+    func receive(message: FeatureMessage, from core: DatadogCoreProtocol) -> Bool {
+        switch message {
+        case .context(let context):
+            return update(context: context)
+        default:
+            return false
+        }
+    }
+
+    /// Adds RUM Error with given message and stack to current RUM View.
+    private func update(context: DatadogContext) -> Bool {
+        guard
+            let tracer = Global.sharedTracer as? Tracer,
+            let integration = tracer.rumIntegration
+        else {
+            return false
+        }
+
+        integration.attribues = context.featuresAttributes["rum"]?.all()
+        return true
+    }
```

Comment:
The method comment here is not clear IMO. It tells about RUM error and RUM view, but nothing in the implementation seems to address this. What do we mean here?

Reply:
Sorry, c/p here :)